### PR TITLE
121 integration

### LINKED ISSUE #121: Integration with Data Visualisation
- [ ] Discuss integration with the visualization team
- [ ] Research what should be done to make the integration possible
- [ ] ? proof of concept

### PR DESCRIPTION
**Pull request by MathiasMeuleman**
Will close #121 

Things have changed :)
e.g. The build folder has been changed to include the analytics project. Also the folders in the build folder have been moved around to create a more logical ordering.
Therefore, the references in the popup and options html pages had to be changed.

!!!IMPORTANT!!!
Before trying it out:
 - from the root folder of this project, run
  `git submodule init && git submodule update --recursive`
 - run ./build.sh clean
 - change the reference in Chrome from `build/octopeer-github` to `build`

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "TI2806"]
```
Comment (mpsijm) [inline .gitmodules]: We probably can't rename their submodule, can we? This name isn't very descriptive. I think, if we leave the link the same, we should be able to edit the name but I'm not sure.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ All documents can in general be found in the doc folder.
 - Use WebStorm (or Git knowledge) to clone the repo to a local folder
 - Run `npm install`
 - Run `npm run-script travis` to check whether everything works correctly
+- Run `cd TI2806 && git submodule init && git pull origin master` to make the analytics work
```
Comment (mpsijm) [inline README.md]: Should be: (tried it and it works) `git submodule init && git submodule update`

File: resources/popup.html [hunk 3]
```diff
@@ -30,13 +30,18 @@
...
-            <div class="col s6">
+        <div class="row">
+            <div class="col s4">
                 <a id="status_toggle" class="btn waves-effect lime waves-light">
                     <i class="material-icons">power_settings_new</i>
                 </a>
             </div>
-            <div class="col s6">
+            <div class="col s4">
+                <a class="btn waves-effect lime waves-light" href="../analytics/index.html?userName=thervh70" target="_blank">
```
Comment (mpsijm) [inline resources/popup.html]: Can the default user be used here? (either indicate `Travis` or leave it empty)

File: .gitmodules (hunk 1 above)
Comment (MathiasMeuleman) [inline .gitmodules]: I am not sure, but we can certainly try :)

File: resources/popup.html [hunk 4]
```diff
@@ -30,13 +30,18 @@
...
+                    <i class="material-icons">timeline</i>
+                </a>
+            </div>
+            <div class="col s4">
```
Comment (mpsijm) [inline resources/popup.html]: Hmm... the right margin looks smaller than the left margin:

File: README.md (hunk 2 above)
Comment (MathiasMeuleman) [inline README.md]: has been changed

File: resources/popup.html (hunk 3 above)
Comment (MathiasMeuleman) [inline resources/popup.html]: yes punt

File: resources/popup.html (hunk 4 above)
Comment (MathiasMeuleman) [inline resources/popup.html]: @thervh70

File: README.md [hunk 5]
```diff
@@ -42,6 +42,7 @@ All documents can in general be found in the doc folder.
 - Use WebStorm (or Git knowledge) to clone the repo to a local folder
 - Run `npm install`
 - Run `npm run-script travis` to check whether everything works correctly
+- Run `git submodule init && git submodule update -recursive` to make the analytics work
```
Comment (mpsijm) [inline README.md]: You have the `-recursive` flag, is this necessary?

Comment (MathiasMeuleman) [inline README.md]: Might not be neccassary right now, but it doesn't hurt and might be needed in the future

Comment (mpsijm) [inline README.md]: Sure, okay :) And the `-` has been changed into a `--` as well now, so I'm good :)

**Comment by mdingjan** [comment]
To be honest, I think that the analytics icon is at least too small.
If got another suggestion for it, which might be more clear: have a look at the following CC image
https://thenounproject.com/term/analytics/29891/

File: .gitmodules (hunk 1 above)
Comment (MathiasMeuleman) [inline .gitmodules]: it works :)

**Comment by thervh70** [comment]
@mdingjan I have added a new icon, it is not the icon you have given me. Because materialize icons are much easier to handle. Please tell me whether you like it.

icons

**Comment by mdingjan** [comment]
@thervh70 I think that's an improvement :)
It's simple and is more clear in my opinion.
Thanks! :D

**Comment by MathiasMeuleman** [comment]
If all feedback has been processed, please merge this issue.

File: README.md [hunk 6]
```diff
@@ -42,6 +42,7 @@ All documents can in general be found in the doc folder.
 - Use WebStorm (or Git knowledge) to clone the repo to a local folder
 - Run `npm install`
 - Run `npm run-script travis` to check whether everything works correctly
+- Run `git submodule init && git submodule update --recursive` to make the analytics work
```
Comment (Exclaminator) [inline README.md]: +1 for updating documentation

**Comment by Exclaminator** [comment]
LGTM, merging

**Pull request merged**